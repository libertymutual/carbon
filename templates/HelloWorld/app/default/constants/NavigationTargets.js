// Navigation Targets
export const CONTACT = 'ContactScreen';
export const HOME = 'HomeScreen';
export const PROFILE = 'ProfileScreen';
export const ABOUT = 'AboutScreen';
export const BIOMETRIC = 'BiometricScreen';
export const LOGIN = 'LoginScreen';
