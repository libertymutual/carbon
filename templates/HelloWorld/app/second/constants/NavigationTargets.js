// Navigation Targets
export const HOME = 'HomeScreen';
export const ABOUT = 'AboutScreen';
