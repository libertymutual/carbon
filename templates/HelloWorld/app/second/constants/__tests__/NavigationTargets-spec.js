import 'react-native';
import React from 'react';
import * as NavigationTargets from '../NavigationTargets';

describe('NavigationTargets tests', () => {

  test('HOME returns correct value', () => {
    expect(NavigationTargets.HOME).toMatchSnapshot();
  });

  test('ABOUT returns correct value', () => {
    expect(NavigationTargets.ABOUT).toMatchSnapshot();
  });
});
