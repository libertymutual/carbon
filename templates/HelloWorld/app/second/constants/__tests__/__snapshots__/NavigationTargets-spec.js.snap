// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationTargets tests ABOUT returns correct value 1`] = `"AboutScreen"`;

exports[`NavigationTargets tests HOME returns correct value 1`] = `"HomeScreen"`;
